cpp/include/cudf_test/column_utilities.hpp

```diff
@@ -46,6 +46,8 @@ namespace detail {
 /**
  * @brief Verifies the property equality of two columns.
  *
+ * This function should not be used directly.
+ *
```

Comment:
Perhaps add a comment here (and similar in the other cases) saying what should be used instead?